core/src/main/java/org/infinispan/topology/LocalTopologyManagerImpl.java

```diff
@@ -813,5 +852,22 @@ CacheTopology getStableTopology() {
    void setStableTopology(CacheTopology stableTopology) {
       this.stableTopology = stableTopology;
       partitionHandlingManager.onTopologyUpdate(currentTopology);
+      if (stableTopology != null) stable.complete(null);
+   }
+
+   List<Address> knownMembers() {
+      return Immutables.immutableListConvert(knownMembers);
+   }
+
+   void setCurrentMembers(List<Address> joiners) {
+      knownMembers = Immutables.immutableSetConvert(joiners);
+   }
+
+   CompletionStage<Void> subscribe() {
```

Comment:
Pick a better name here, subscribe means something else for reactive processing.